docker/Dockerfile

```diff
@@ -17,7 +17,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # Add Postgres PPA
 ARG postgresql_gpg_key=B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys "${postgresql_gpg_key}" && \
+RUN mkdir -p /root/.gnupg && chmod 700 /root/.gnupg && \
+    gpg --recv-keys --no-default-keyring --keyring /tmp/pgdg.key --keyserver pgp.mit.edu --recv-keys "${postgresql_gpg_key}" && \
+    gpg --no-default-keyring --keyring /tmp/pgdg.key --export "${postgresql_gpg_key}" > /etc/apt/trusted.gpg.d/pgdg.gpg && \
```

Comment:
`apt-key adv` borks when Ubuntu 18.04 runs in Docker's `buildx` - using `gpg` to achieve same result